bin/main.ml

```diff
@@ -38,12 +39,16 @@ let redemon paths verbose command args =
                   ignore (Luv.FS_event.stop watcher);
                   Luv.Handle.close watcher stop_program
               | Ok (file, events) ->
-                  stop_program () |> start_program;
                   if verbose then (
-                    print_endline "Stopping and starting the program.";
                     if List.mem `RENAME events then prerr_string "renamed ";
                     if List.mem `CHANGE events then prerr_string "changed ";
-                    prerr_endline file )))
+                    prerr_endline file );
```

Comment:
you should probably consider calling `flush stderr`. That could explain why folks reported messages not showing up.

Reply:
Doesn't prerr_endline flush it already?